hodl/exchanges/PoloniexHelperAPI.py

```diff
@@ -5,14 +5,9 @@
 """
 import sys
 import time
-
+import datetime
 from poloniex import Poloniex
-
 from hodl.ConfRetriever import ConfRetriever
-
-
-import datetime
-
```

Comment:
The spacing above was not an issue. Instead re-read the PEP8 link I sent you (see previous comment). Specifically the ordering of the imports was the issue. `datetime` should be grouped (in the first group) with other standard libraries such as `sys` and `time`.